Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-reco
         pkg-config \
         build-essential \
         # only relevant when using editable/github dependencies, which is discouraged
-        # git \
+        git \
```

Comment:
This should be reverted whenever a new release happens for `commonground-api-common`/ OAF.